src/config.rs

```diff
@@ -133,7 +135,11 @@ pub fn parse_config() -> Result<Config, Error> {
         )?
         .0,
 
-        admin_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 15021),
+        // admin API should only be accessible over localhost
+        // TODO: Unix domain socket?
+        admin_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::LOCALHOST), 15000),
+        readiness_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 15021),
+        stats_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 15020),
```

Comment:
I don't think this is used yet.

Reply:
Ah good catch; I can remove it unless you want to use it for your stats work

Reply:
Removing for now